includes/admin/helpers/functions/layout.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Admin settings helper functions
+ *
+ * @since x.x
+ */
+
+/**
+ * Prints out all settings sections added to a particular settings page.
+ *
+ * @link https://developer.wordpress.org/reference/functions/do_settings_sections/
+ *
+ * @param string $page
+ *
+ * @since x.x
+ */
+function wpbdp_admin_do_settings_sections( $page ) {
```

Comment:
Looks like these functions should be in a class. Maybe the admin pages class would make sense?

Reply:
Agreed :)